appendix-basic65.tex

```diff
@@ -5860,12 +5857,11 @@ \subsection{NEXT}
 \item [Remarks:] The {\bf index} variable after {\bf NEXT} is
                optional. If it is omitted, the variable
                for the current loop is assumed.
-               % i dont understand this part ^^ - impakt
                Several consecutive {\bf NEXT} statements may be
                combined by specifying the indexes in a comma
                separated list. The statements
-               {\bf NEXT I:NEXT J:NEXT K} and
-               {\bf NEXT I,J,K} are equivalent.
+               \screentext{NEXT I,NEXT J,NEXT K} and
```

Comment:
The commas you switched to should be reverted back to colons (as `NEXT I, NEXT J, NEXT K` would produce a syntax error)

Reply:
oops